massa-graph/src/block_graph.rs

```diff
@@ -1188,7 +1204,28 @@ impl BlockGraph {
     pub fn get_export_block_status(&self, block_id: &BlockId) -> Option<ExportBlockStatus> {
         self.block_statuses
             .get(block_id)
-            .map(|block_status| block_status.into())
+            .map(|block_status| match block_status {
+                BlockStatus::Incoming(_) => ExportBlockStatus::Incoming,
+                BlockStatus::WaitingForSlot(_) => ExportBlockStatus::WaitingForSlot,
+                BlockStatus::WaitingForDependencies { .. } => {
+                    ExportBlockStatus::WaitingForDependencies
+                }
+                BlockStatus::Active(active_block) => {
+                    let block = self
+                        .storage
+                        .retrieve_block(block_id)
+                        .expect("Missing block in storage.");
```

Comment:
Why not return None here to avoid crash ?

Reply:
Because it's an inconsistency, this should not happen